libs/wingsdk/src/target-sim/api.inflight.ts

```diff
@@ -44,6 +44,7 @@ interface StateFileContents {
 
 interface ApiRouteWithFunctionHandle extends ApiRoute {
   functionHandle: string;
+  expressRouteHandle: any;
```

Comment:
Since this is internal, could it use the actual Layer type from express without getting yelled at by JSII?

Reply:
Also should this be expressLayer instead?

Reply:
Renamed. Unfortunately I couldn't find an exposed type for Layer in Express 4.17